.env.example

```
@@ -138,3 +138,21 @@ INGESTION_CRON_SECRET=your_cron_secret_here
 # Get credentials: https://ads.tiktok.com/marketing_api/docs
 # JOVIE_TIKTOK_PIXEL_ID=your_tiktok_pixel_code
 # JOVIE_TIKTOK_ACCESS_TOKEN=your_tiktok_access_token
+
+# ===== Additional (previously undocumented) =====
+# Set to 1 only in local E2E / browse sessions. Leave blank in production.
+NEXT_PUBLIC_E2E_MODE=
+# Auto-populated by CI with the git SHA of the build. Leave blank locally.
+NEXT_PUBLIC_BUILD_SHA=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_BUILD_SHA key should go before the NEXT_PUBLIC_E2E_MODE key

(UnorderedKey)